tutorials/AgileRL/agilerl_matd3.py

```diff
@@ -21,7 +21,7 @@
     # Define the network configuration
     NET_CONFIG = {
         "arch": "mlp",  # Network architecture
-        "h_size": [32, 32],  # Actor hidden size
+        "hidden_size": [32, 32],  # Actor hidden size
```

Comment:
Minor but very bizarre that the original names were used, feel like it’s so unintuitive to have h_size or whatnot, especially for new users. Good to change it to be readable.